hack/tools/.srl

```diff
@@ -0,0 +1 @@
+E72C58454F0D02E0
```

Comment:
What is this?

Reply:
No idea. It got added during one of the merges

Reply:
It is not in the master branch. I guess this was auto-generated at some stage, but there is no need to commit this file.